common/BasicTypes.asn1

```diff
@@ -25,13 +25,148 @@ BasicTypes DEFINITIONS AUTOMATIC TAGS ::= BEGIN
 
 EXPORTS ALL;
 
--- TODO PTC PFC!
+PUSC-BOOL1 ::= BOOLEAN --PFC-0
```

Comment:
1. do not duplicate comment in ACN
2. if you add PFC, maybe it would be good to include PTC somewhere